src/compiler/worker/getConfig.js

```diff
@@ -14,10 +14,15 @@ module.exports = function getConfig(
   configOptions: any,
   platform: Platform
 ) {
-  return makeReactNativeConfig(
+  let config;
+  try {
     // $FlowFixMe
-    require(configPath),
-    configOptions,
-    platform
-  );
+    config = require(configPath);
+  } catch (e) {
+    throw new Error(
```

Comment:
We should default to the following config when none is present:

```js
import { createWebpackConfig } from 'haul';
 
export default {
  webpack: createWebpackConfig({
    entry: './index.js',
  }),
};
```

So we should do something like this:
1. Check if the config file is present, if not, use default config
2. If config file is present, try to load it

I'm not sure if we should be catching the error and throwing a new error because it'll shadow the original error which caused the `require` to fail, such as syntax error, which will make it harder to figure out how to fix the issue.

Reply:
Also, should it worth pointing out that we fallback to default config?

Reply:
@Krizzu we could show a log when starting haul. Something like: "No configuration file found. Using default configuration."

Reply:
Won't work properly, since it's used by worker, and the UI is a react app. The log won't be visible

Reply:
Yeah, agree that we should fall back to the default configuration I wasn't aware that it's possible. But I like the idea. 👍